src/masternodes/vault.h

```diff
@@ -160,6 +160,7 @@ class CVaultView : public virtual CStorageView
     struct VaultKey         { static constexpr uint8_t prefix() { return 0x20; } };
     struct OwnerVaultKey    { static constexpr uint8_t prefix() { return 0x25; } };
     struct CollateralKey    { static constexpr uint8_t prefix() { return 0x21; } };
+    struct LoanKey          { static constexpr uint8_t prefix() { return 0x21; } };
```

Comment:
Add it to `CheckPrefixes` in `masternodes.h` it will fails to compile since it duplicates `CollateralKey` it should 0x26.